gclient/tests/upload.rs

```diff
@@ -27,7 +27,11 @@ const PATHS: [&str; 2] = [
     "../target/wat-examples/inf_recursion.wasm",
 ];
 
-async fn upload_programs_and_check(api: &GearApi, codes: Vec<Vec<u8>>) -> Result<()> {
+async fn upload_programs_and_check(
+    api: &GearApi,
+    codes: Vec<Vec<u8>>,
+    timeout: Option<Duration>,
```

Comment:
I'd add milliseconds and construct Duration type inside

Reply:
Actually we can do this when we will heave need =) currently cannot see any points